stake-pool/program/proptest-regressions/state.txt

```diff
@@ -1 +1,2 @@
 cc 41ce1c46341336993e5e5d5aa94c30865e63f9e00d31d397aef88ec79fc312ca
+cc 2e5aed24877ce7b166d018e73d8617f51ec250b8ef51a9f539a978ef99b3304b # shrinks to test_amount = 1
```

Comment:
is this needed?

Reply:
not sure what is or isn't needed when it comes to proptests actually...